### DISCUSSION TITLE
How do I train with a lot of random data for each x position?

### DISCUSSION THREAD

**Discussion by tangri** [lululxvi/deepxde#1778] [Q&A]
I have a lot of random data for each x position.
I have several measurements of data at each location as if the experiment was repeated N times 

Currently, I am using:

     observe_x = np.vstack((np.linspace(-1, 1, num=500), np.full((500), 1))).T
     observe_y = dde.icbc.PointSetBC(observe_x, funcr(observe_x), component=0)

But this only allows one unique measurement at each x. How can I have more than 1 noisy value for observe_y for each observe_x?

**Comment by lululxvi** [comment]
It is the same: just use `dde.icbc.PointSetBC`.
